open-machine-learning-jupyter-book/ml-fundamentals/parameter-optimization/parameter-optimization.ipynb

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
<p>should add the descendants <code></code>`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
DONE